Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -590,5 +590,84 @@ class FeaturevisorInstanceTests: XCTestCase {
         XCTAssertTrue(refreshed)
         XCTAssertTrue(updatedViaOption)
     }
+    
+    func testShouldStartRefreshing() {
+
+        // GIVEN
+        var revision = 1
+        var refreshedCount = 0
+        let refreshInterval = 1
+        let expectedRefreshCount = 3
+        let maxWaitTime = TimeInterval(expectedRefreshCount) * TimeInterval(refreshInterval) + 1
```

Comment:
I would rather go with mocking the timer and mock the "time". Example - https://medium.com/@isahkaren16/testing-timer-based-features-in-swift-5ecaa4bb0c73

Anyway, if it's too complicated for now, lets skip for now and back to this later